mne_bids/write.py

```diff
@@ -193,11 +197,32 @@ def _channels_tsv(raw, fname, overwrite=False):
         ch_type.append(map_chs[_channel_type])
         description.append(map_desc[_channel_type])
     low_cutoff, high_cutoff = (raw.info["highpass"], raw.info["lowpass"])
-    if raw._orig_units:
+    # If data are being *converted*, unit is determined by destination format:
+    #   - `eeglabio.raw.export_set` always converts V to µV, cf:
+    # https://github.com/jackz314/eeglabio/blob/3961bb29daf082767ea44e7c7d9da2df10971c37/eeglabio/raw.py#L57
+    #
+    #   - `mne.export._edf_bdf._export_raw_edf_bdf` always converts V to µV, cf:
+    # https://github.com/mne-tools/mne-python/blob/1b921f4af5154bad40202d87428a2583ef896a00/mne/export/_edf_bdf.py#L61-L63
+    #
+    #   - `pybv.write_brainvision` converts V to µV by default (and we don't alter that)
+    # https://github.com/bids-standard/pybv/blob/2832c80ee00d12990a8c79f12c843c0d4ddc825b/pybv/io.py#L40
+    # https://github.com/mne-tools/mne-bids/blob/1e0a96e132fc904ba856d42beaa9ddddb985f1ed/mne_bids/write.py#L1279-L1280
+    if convert_fmt:
+        volt_like = "µV" if convert_fmt in ("BrainVision", "EDF", "EEGLAB") else "V"
```

Comment:
this logic implies that all file formats except bv, edf and eeglab store voltages in `V`.
is this correct? 
or is it rather correct, that all other formats dont do any conversion? then we would still need to access `orig_unit` here

maybe practically irrelevant if `mne_bids` only supports these export formats (for eeg they are currently the only formats allowed in BIDS), but couldnt there be MEG formats that store both MEG and EEG in the same file (like curry does)?

Reply:
Also the user can request/force conversion in spite of what the bids standard requires